### DISCUSSION TITLE
Documentation missing "domain" as a required string

### DISCUSSION THREAD

**Discussion by chandru89new** [vercel/community#1072] [Idea]
<img width="823" alt="image" src="https://user-images.githubusercontent.com/10336112/202898227-667367e5-3996-4998-acba-902dbed528ff.png">

I think "domain" is missing in the Path Parameters.

**Comment by amyegan** [comment]
Thanks for pointing this out @chandru89new! I opened a new ticket to get this fixed 🙂
